cppcheck_junit.py

```diff
@@ -46,6 +46,8 @@ def parse_arguments() -> argparse.Namespace:
     )
     parser.add_argument("input_file", type=str, help="Cppcheck XML version 2 stderr file.")
     parser.add_argument("output_file", type=str, help="JUnit XML output file.")
+    parser.add_argument("error_exitcode", type=int, nargs="?", const=0, help="If errors are found, \
```

Comment:
Prefer not to use `\` on strings. In Python you can simply do something like:

```py
("my string broken "
"across two lines.")
```

Reply:
Good suggestion. I will change it after we agreed on the content of the string.

Reply:
Made a fixup commit.